src/control/controllers/controllers/node_reactive_vector_control.py

```diff
@@ -327,6 +333,8 @@ def callback(self, cone_msg: ConeDetectionStamped):
             if orderSpline > 3:
                 orderSpline = 3
 
+            orderSpline = 2  # track hard code linear quick fix
+
```

Comment:
This looks like it could cause issues @b1n-ch1kn

Reply:
this whole controller is a work in progress, but i think its probably worth keeping so at least we know the direction we tried to go